site/content/docs/concepts/pipelines.md

```diff
@@ -93,27 +93,31 @@ stages:
 
 There are 3 main parts of this file: the `name` field, which is the name of your CodePipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must run `copilot pipeline update` afterwards.
 
-Typically, you'll update this file if you add new environments you want to deploy to, or want to track a different branch.
+Typically, you'll update this file if you add new environments you want to deploy to, or want to track a different branch. The pipeline manifest is also where you may add a manual approval step before deployment or commands to run tests (see "Adding Tests," below) after deployment.
 
 ### Step 3: Updating the Buildspec (optional)
 
 Along with `pipeline.yml`, the `pipeline init` command also generated a `buildspec.yml` file in the `copilot/` directory. This contains the instructions for building and publishing your service. If you want to run any additional commands, besides `docker build`, such as unit tests or style checkers, feel free to add them to the buildspec's `build` phase.
 
 When this buildspec runs, it pulls down the version of Copilot which was used when you ran `pipeline init`, to ensure backwards compatibility.
 
-### Step 4: Creating your Pipeline
+### Step 4: Pushing New Files to your Repository
 
-Now that your `pipeline.yml` and `buildspec.yml` are created, check them in and push them to your repository. The `buildspec.yml` is needed for your pipeline's `build` stage to run successfully. Once you've done that, to actually create your pipeline run:
+Now that your `pipeline.yml`, `buildspec.yml`, and `.workspace` files have been created, add them to your repository. The entire `copilot/` directory is needed for your pipeline's `build` stage to run successfully. 
```

Comment:
I am wondering If we need everything under the `copilot/` directory, should we also modify the instructions in "Creating a Pipeline in 3 steps"? 

It seems a little unclear to me whether we just need `pipeline.yml`, `buildspec.yml` and `.workspace` (since in the 3 steps instructions it only included these files), or we need the entire directory (as mentioned here)

Reply:
Ah, good question! Will fix to make more specific!